src/app/i18n/de.json

```diff
@@ -9,8 +9,8 @@
     "PHRASE_NO_BUILDS": "Keine Konfigurationen zum Vergleich ausgewählt!",
     "PHRASE_NO_RETROCH": "Keine Umrüständerungen",
     "PHRASE_SELECT_BUILDS": "Ausstattung zum Vergleich auswählen",
-    "PHRASE_SG_RECHARGE": "Zeit von 50% bis 100% der Ladung bei vollem SYS Kondensator",
-    "PHRASE_SG_RECOVER": "Erneuerung (zu 50%) nach Zusammenbruch bei vollem SYS Kondensator",
+    "PHRASE_SG_RECHARGE": "Zeit von 50% bis 100% der Schildladung mit 4 Pips auf System",

```

Comment:
The german description for the controls in-game says "Leitet die Energie vom Energieverteiler zu den Schiffssystemen um. ..."
So we can translate it with:
"Zeit von 50% bis 100% der Schildladung mit 4 Energieverteilerpunkten auf System" or a little shorter "... 4 Energiepunkten auf System" or instead of 4 just "... vollen Energiepunkten auf System".
The SYS capacitator is called "System" in german. The others are "Antrieb" and "Waffen".

Reply:
Yeah, I'm German as well 😉 I don't have a problem with wording. But your translation now misses that we assume the capacitor to be full to start with. Not only do we assume to have 4 pips to sys but also that it is full. "Voll" here really was talking about the capacitor, not the pips.

Reply:
Ah. Then I misinterpreted the sentence. I thought the shield recharges faster, if i got all pips on SYS but you mean the capacitor has to be full, regardless of the pips on system. The I think we can leave the old translation or mabye change "SYS Kondensator" to "System-Kondensator".

Reply:
The shield only regenerates faster with more pips to SYS if your SYS recharge rate is lower than the energy drain of the shield itself. As soon as you have enough pips to SYS such that you regen more energy than is drained by the shield, the pips do not matter anymore.

Furthermore, I just double checked how regen time works on coriolis and found out that your translation is false. The time is calculated based on the current pips settings. So it should just say something like: `"Zeit von 50% bis 100% der Schildladung bei anfänglich vollem System-Kondensator"`

Reply:
I have corrected the translations. Thanks for your feedback.